packages/api/src/services/repository/repository.service.ts

```diff
@@ -19,8 +19,8 @@ export class RepositoryService implements Service {
 
   constructor() {
     this.repositoryRestService = ServiceProvider.get(RepositoryRestService);
-    this.repositoryListMapper = MapperProvider.get(RepositoryListMapper);
-    this.repositorySizeInfoMapper = MapperProvider.get(RepositorySizeInfoMapper);
+    this.repositoryListMapper = new RepositoryListMapper();
+    this.repositorySizeInfoMapper = new RepositorySizeInfoMapper();
```

Comment:
Additionally you can try to make the code more consistent throughout the codebase during this refactoring task instead of just replacing the `MapperProvider.get` with `new`. This is the meaning of the boy scout rule.
For example, you can make all mapper instantiations to happen directly as a field.